modules/ROOT/pages/forum-moderation.adoc

```diff
@@ -4,6 +4,78 @@ include::_attributes.adoc[]
+
+// Will the pre-made community in the workshop have various posts, including content that matches various criteria to be held for review, and users that were frozen for posting too often or similar criteria?
+
+To moderate forums, log in, select a community, and select *Moderation*.
+
+The *Home* tab shows these options:
+
+* *Pending Discussions*: Respond to and moderate new questions and discussions.
+* *Flagged Files*: Unflag or delete files flagged as suspicious.
+* *Flagged Direct Messages*: Unflag or delete direct messages flagged as suspicious.
+* *Flagged Private Messages*: Unflag or delete private messages flagged as suspicious.
+* *Frozen Members*: Unfreeze user accounts.
+
+The *Moderate* tab shows these options:
+
+* *Pending Discussions*: Respond to and moderate new questions and discussions.
+* *Flagged Files*: Unflag or delete files flagged as suspicious.
+* *Flagged Messages*: Unflag or delete messages flagged as suspicious.
+* *Frozen Members*: Unfreeze user accounts.
+* *Audit Trail*: Review an audit trail of all user activity on the forums.
+
+The *Rules* tab shows these options:
+
+* *Moderation Rules*: View, add, and edit rules to moderate member-generated content. For example, members who post or comment too frequently can be frozen, the first post or comment made by a new member can be held for moderator review, and posts or comments with banned keywords can be blocked, flagged, held for review, or edited so that the banned words are replaced with asterisks.
+* *Content Criteria*: Filter content with criteria such as lists of keywords to block. For example, you might have one list of terms found in financial scam posts, and another list of terms used as offensive insults.
+* *Member Criteria*: Filter members with criteria such as user creation date, whether the user has contributed to the community before, and user profiles such as customers and partners.
+
+== Moderating with Community Workspaces
+
+Community administrators may either use the same forum pages that are seen by external users to administer and post on the forums, or use Community Workspaces pages in Salesforce to administer and post on the forums and to access many other options for monitoring and managing communities.
+
+To use Community Workspaces, go to your list of communities and choose the *Workspaces* link for the community.
+
+This Community Workspaces interface for your community shows options to open the *Builder*, *Moderation*, *Content Management*, and *Dashboards*.
+
+There is also a customizable dashboard at the top with default items showing *Members*, *New Members*, *Unanswered Questions*, and *Flagged Discussions*.
```

Comment:
I would make each of these things an H3 to make it stand out more.
Instead of Choose *Dashboards*, have Dashboards as the heading title. 
This is just a suggestion.

Reply:
I'm not exactly sure the best way to do this, so I'm postponing for now, but I would like to revisit this idea later when we have time, if you remember, please.